src/xchain/modules/folks-gmp.ts

```diff
@@ -145,6 +145,7 @@ export const prepare = {
         message,
         extraArgsParams,
         payload,
+        adapterId,
```

Comment:
This is incorrect again, you are overriding with the adapter id of what was used to send the message from the spoke to hub. My suggestion is to keep it how it was before this PR